packages/react-draggable-dialog/src/components/DraggableDialog/DraggableDialog.test.tsx

```diff
@@ -1,16 +1,183 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import { act, render, renderHook } from '@testing-library/react';
+
 import { DraggableDialog } from './DraggableDialog';
+import { useDraggableDialog } from './useDraggableDialog';
+import { DragEndEvent } from '@dnd-kit/core';
+
+jest.mock('@fluentui/react-components', () => ({
+  ...jest.requireActual('@fluentui/react-components'),
```

Comment:
similar questions as in this PR 
https://github.com/microsoft/fluentui-contrib/pull/109/files#r1497893621
https://github.com/microsoft/fluentui-contrib/pull/109/files#r1497886118

Reply:
Makes sense. I'll address everything in the #109 PR and reflect the changes here, if necessary.